site/public/css/colors.css

```diff
@@ -23,6 +23,7 @@
     --hover-light-border-actionable-blue: #2e6da4;
     --hover-border-actionable-blue: #204d74;
     --hover-dark-border-actionable-blue: #122b40;
+    --hover-notification: #dfdfdf;
```

Comment:
Do we really need more colors?  Are we sure there isn't an existing color which is close enough?